### PR TITLE
Update README.md (Some small spelling stuffs)

### PR DESCRIPTION
**Pull request by aismann**
Same small spelling stuffs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 * Set default C++ standard to 14
 * Set min deploy target ios sdk to 9.0
 * Remove tinyxml2, use more fast pugixml instead
-* Use curl for HttpClient,Downloader on all platforms
+* Use curl for HttpClient downloader on all platforms
```
[the remark this answers is not in the document]
Comment (aismann) [reply README.md:28]: Better write as follow:
Using curl for transferring data with URL syntax.

**Comment by aismann** [comment]
And thus ist also better:
Using pugixml

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (halx99) [reply README.md:28]: agree

**Comment by halx99** [comment]
Merge first, thanks

**Pull request merged**